### ISSUE TITLE
Install Error: Visual Studio MPF 15.0 not present

### ISSUE THREAD

**Issue by drichardson** [Kir-Antipov/LineSorter#5]
On one of my computers, I've successfully installed LineSorter on VS 2019. However, on my laptop, with Visual Studio 2019 installed with the Game Development packages checked (and everything else unchecked), I get an error when I attempt to install LineSorter that says:

*The following references must be present before installation can continue: Visual Studio MPF 15.0*.

![image](https://user-images.githubusercontent.com/36176/57199736-3683c300-6f37-11e9-93b4-3ed7157f724f.png)

**Comment by Kira-NT** [comment]
I'm sorry about that!

VS2019 uses MPF 16.0, and I completely forgot to specify in the extension's manifest that this version is also supported)

Give me 15 minutes and I'll fix it)

**Comment by Kira-NT** [comment]
Just released an update
Is the problem solved?

**Comment by drichardson** [comment]
Yes, I think.

I first tried installing LineSorter from the Extensions window in VS2019, but that didn't work (I got the same error) even though it told me it was trying to use LineSorter 4.6.

But then when I downloaded and installed it from marketplace (https://marketplace.visualstudio.com/items?itemName=KirAntipov.LineSorterByKir) it worked fine.

Perhaps the VS2019 Extension manager was using an older cached version? I'm not sure but I think the fix worked.

Thanks for addressing this so quickly!
